main.py

```diff
@@ -11,46 +11,37 @@
         `brew install swig`
 """
 import fitz
-import os
-
+from pathlib import Path
 
 # configure the path to your stationary
 STATIONARY =    "Briefkopf/Briefkopf.pdf"
+BACKSIDE_CT =   "Briefkopf/CT_Rückseite.pdf"
+BACKSIDE_MRT =  "Briefkopf/MRT_Rückseite.pdf"
 DOC_FOLDER =    "Dokumente/"
 OUTPUT_FOLDER = "PDF/"
 
+content_files = [(path, Path(str(path).replace(DOC_FOLDER, OUTPUT_FOLDER))) for path in Path(DOC_FOLDER).glob("**/*.pdf")]
 
-content_files = []
-output_dirs = [OUTPUT_FOLDER] # add base dir to list
-
-# retrieve all *.pdf files and directories/subdirectories from DOC_FOLDER
-for root, dirs, files in os.walk(DOC_FOLDER):
-    for name in dirs:
-        output_dirs.append(str.replace(os.path.join(root, name), DOC_FOLDER, OUTPUT_FOLDER))
-    for name in files:
-        if name.lower().endswith(".pdf"):
-            content_files.append(os.path.join(root, name))
-
-print(content_files)
-print(output_dirs)
-
-# create missing folders from list
-for directory in output_dirs:
-    if not os.path.exists(directory):
-        os.mkdir(directory)
+stat = fitz.open(STATIONARY)
+backside_CT = fitz.open(BACKSIDE_CT)
+backside_MRT = fitz.open(BACKSIDE_MRT)
 
 # iterate files and merge
-for content_file in content_files:
+for input_path, output_path in content_files:
+    output_path.parent.mkdir(parents=True, exist_ok=True)
     
-    cont = fitz.open(content_file)
-    stat = fitz.open(STATIONARY)
+    cont = fitz.open(input_path)
 
     page = cont.load_page(0)
     page_front = fitz.open()
     page_front.insert_pdf(stat, from_page=0, to_page=0)
     page.show_pdf_page(page.rect, page_front, pno=0, keep_proportion=True, overlay=False, oc=0, rotate=0, clip=None)
 
-    # output file name and dir
-    result_file = content_file.replace(DOC_FOLDER, OUTPUT_FOLDER)
+    # check if files need 2nd page. Needs workaround with byte-decoding because pathlib uses
+    # different byte decoding of umlaut than python strings
+    if input_path.stem.startswith(b'Aufkla\xcc\x88rung_CT'.decode()):
```

Comment:
this appends the "second page" to all `Aufklärung_CT*` and `Aufklärung_MRT`, thus appending it to `*_Punktion_Drainage` etc.

Reply:
Ok, get it. Needs further workaround

Reply:
> Umlaut workaround does not work on windows. Actually not needed -_-

match with regex?

Reply:
Untested:
```py
if re.match(r"^Aufkl.rung_CT", input_path.stem)
```